### PR TITLE
Kevin/404

### PR DESCRIPTION
**Pull request by thekevinscott**
Questions/comments:

* Do we have integration tests? If so, should I add one for a 404 page?
* I believe in a `<Switch />` we can get away with just `<Route component={pages.MissingPage404} />` but correct me if wrong
* How do we feel about flexbox in CSS?
* I ran into a type issue out of the gate and a parseInt missing radix error. I've fixed those in the branch but if there's some other way around those errors let me know.

How much fidelity are we looking for in the translation from Invision to code? For instance, I notice the headers differ between the App and Invision, and the font weights specified don't seem to be there (looks like it's just `normal` and `bold`):

In code:
screen shot 2018-07-23 at 19 59 52

In Invision:
screen shot 2018-07-23 at 20 00 47

### CODE REVIEW

**Comment by thekevinscott** [comment]
I've updated this PR to not rely on flexbox.

I'm not sure the best way to get true vertical centering without it and would love input. It looks like the parent containers are not set to 100%, so I hard coded a margin on the `h1` that works reasonably well for iphone 6/7/8 in Chrome. But it won't always be centered for every layout, let alone tablet/desktop.

Thoughts?

**Comment by andrewtremblay** [comment]
@thekevinscott Unless Becca told you otherwise I don't think we need true vertical centering here, just adequate margins to display the 404 in a generally central place. As for non-flexbox solutions: `transform: translateY(-50%);` gets you most of the way towards vertical centering (though mileage may vary). 

As for adhering to the designs, I've kept it close as possible but it doesn't have to be religiously followed. For example, new font weights, (when found), should be added, but pixel perfection should be a low priority.  A final evaluation before it goes live will probably be done by Stacy and/or Becca, that's when the pixels can be pushed around.

File: frontend/its-our-call/src/pages/IssueView.tsx [hunk 1]
```diff
@@ -132,9 +132,9 @@ class IssueView extends React.Component<IssueViewProps, IssueViewState> {
     let requiresCloture = hashed.requiresThreeFifth ? hashed.requiresThreeFifth === 'true' : false;
     let adjustForTiebreaker = hashed.adjustForTiebreaker ? hashed.adjustForTiebreaker === 'true' : true;
     // ^ simple or three-fifths majority
-    let percentYea = hashed.percentYea ? Number.parseInt(hashed.percentYea) : 55;
-    let percentNay = hashed.percentNay ? Number.parseInt(hashed.percentNay) : 40;
-    let percentUncommitted = hashed.percentUncommitted ? Number.parseInt(hashed.percentUncommitted) : 5;
+    let percentYea = hashed.percentYea ? Number.parseInt(hashed.percentYea, 10) : 55;
```
Comment (andrewtremblay) [inline frontend/its-our-call/src/pages/IssueView.tsx:135]: Good catch here, I forgot that the radix param is not 10 by default in all implementations.

File: frontend/its-our-call/src/pages/MissingPage404.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.MissingPage404 {
+  font-family: 'Montserrat', sans-serif;
```
Comment (andrewtremblay) [inline frontend/its-our-call/src/pages/MissingPage404.css]: font-family is already defined in App.css, not sure if it's needed again here.

File: frontend/its-our-call/src/pages/MissingPage404.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  font-size: 26px;
```
Comment (andrewtremblay) [inline frontend/its-our-call/src/pages/MissingPage404.css]: We've been using relative units (mostly `rem`) as the primary units, with minimal need for `px`. I have a similar opinion to the writer of this gist: https://gist.github.com/basham/2175a16ab7c60ce8e001 (However, I can probably be convinced away from this if there's some special case here.)

File: frontend/its-our-call/src/@types/react-tiny-popover/index.d.ts [hunk 4]
```diff
@@ -54,7 +54,7 @@ declare module 'react-tiny-popover' {
       style?: React.CSSProperties;
       arrowSize?: number;
       // tslint:disable-next-line:no-any
-      arrowColor?: React.CSSWideKeyword | any;
```
Comment (andrewtremblay) [inline frontend/its-our-call/src/@types/react-tiny-popover/index.d.ts:57]: Ah, you caught my scratchwork, committed accidentally.

**Review by andrewtremblay — CHANGES_REQUESTED** [review]
LG2me, just change the css units to relative ones (or convince me otherwise).

**Comment by thekevinscott** [comment]
My apologies for taking so long to address these comments! 

* font-family is removed, you're right that it was not necessary
* I'm not sure if there's standard definitions for the various `rem` values? I've eyeballed it to be equivalent to `26px` but feel free to change if I screwed this up

**Pull request closed without merging**